cluster/certificates.go

```diff
@@ -20,6 +20,16 @@ func SetUpAuthentication(ctx context.Context, kubeCluster, currentCluster *Clust
 		if currentCluster != nil {
 			kubeCluster.Certificates = currentCluster.Certificates
 		} else {
+			logrus.Infof("[certificates] Attempting to recover incomplete deployment")
```

Comment:
Can this log be clearer like "Attempting to recover certificates from backup dir on host %s" also can you use log.Infof with ctx